lib/runner/variant-reassignment.js

```diff
@@ -373,9 +496,10 @@ export default class VariantReassignment {
     return false
   }
 
-  async _removeVariantsFromCtpProductToUpdate (anonymizedProductDraft, ctpProductToUpdate) {
+  _removeVariantsFromCtpProductToUpdate (anonymizedProductDraft, ctpProductToUpdate) {
     const skusToRemove = this.productService.getProductDraftSkus(anonymizedProductDraft)
-    await this.productService.removeVariantsFromProduct(ctpProductToUpdate, skusToRemove)
+    this.logger.debug('Removing %d variants from ctpProductToUpdate', skusToRemove.length)
+    return this.productService.removeVariantsFromProduct(ctpProductToUpdate, skusToRemove)
```

Comment:
I'd rather put async/await here as we don't really return anything in this method? In sync JS, you also don't always return last statement, but you return when you need it.

Reply:
updated bea02bf